concent_api/core/tests/test_core_views.py

```diff
@@ -473,10 +496,11 @@ def test_send_should_return_http_202_if_task_to_compute_deadline_is_correct(self
 class CoreViewReceiveTest(TestCase):
 
     def setUp(self):
-        self.compute_task_def = message.ComputeTaskDef()
-        self.compute_task_def['task_id'] = '1'
-        self.compute_task_def['deadline'] = int(dateutil.parser.parse("2017-11-17 10:37:00").timestamp())
         with freeze_time("2017-11-17 10:00:00"):
+            self.compute_task_def               = message.ComputeTaskDef()
+            self.compute_task_def['task_id']    = '1'
+            self.compute_task_def['subtask_id'] = '1'
+            self.compute_task_def['deadline']   = get_current_utc_timestamp() + 2220
```

Comment:
It will be more readable if you write this as 
```
self.compute_task_def['deadline']   = get_current_utc_timestamp() + (60 * 37)
```

Reply:
:+1: